### ISSUE TITLE
Bad way to Clear 

### ISSUE THREAD

**Issue by Eruptsy** [RetroPackets/Email-Invader#1]
This is a bad way to clear and makes this only supported on linux. There a way to clear screen with ansi escape sequences https://github.com/RetroPackets/Email-Invader/blob/379a66ba5e28f76296d5fc0dd8e04f34d06b4dd7/invader.py#L16

**Comment by RetroPackets** [comment]
Typing clear is used to clear your terminal all the time idiot

**Comment by Eruptsy** [comment]
I dont think you understood what i said. You're script only clears the screen on LINUX which makes your script compatible only with linux. What i said is, there is another way to clear screen for both windows and linux.

**Comment by RetroPackets** [comment]
I make linux too you idiot
